app/services/clients/adminusers.client.js

```diff
@@ -276,116 +199,84 @@ module.exports = function (clientOptions = {}) {
+    this.client = new Client(SERVICE_NAME)
+    configureClient(this.client, url)
+    const body = {
+      email: invitee,
+      sender: senderId,
+      service_external_id: serviceExternalId,
+      role_name: roleName
+    }
+    const response = await this.client.post(url, body, 'invite a user to join a service')
+    return response.data
   }
 
   /**
    * Get a invited users for a given service
    * @param serviceExternalId
    */
-  function getInvitedUsersList (serviceExternalId) {
-    return baseClient.get(
-      {
-        baseUrl,
-        url: `/v1/api/invites`,
-        qs: {
-          serviceId: serviceExternalId
-        },
-        json: true,
-        method: 'GET',
-        description: 'get invited users for a service',
-        service: SERVICE_NAME
-      }
-    )
+  async function getInvitedUsersList (serviceExternalId) {
+    const url = `${baseUrl}/v1/api/invites?serviceId=${serviceExternalId}`
+    this.client = new Client(SERVICE_NAME)
+    configureClient(this.client, url)
+    const response = await this.client.get(url, 'get invited users for a service')
+    return response.data
   }
 
   /**
    * Get a valid invite or error if it's expired
    * @param inviteCode
    */
-  function getValidatedInvite (inviteCode) {
-    return baseClient.get(
-      {
-        baseUrl,
-        url: `/v1/api/invites/${inviteCode}`,
-        json: true,
-        description: 'find a validated invitation',
-        service: SERVICE_NAME
-      }
-    )
-  }
-
-  function updateInvitePassword (inviteCode, password) {
-    return baseClient.patch(
-      {
-        baseUrl,
-        url: `/v1/api/invites/${inviteCode}`,
-        json: true,
-        body: [{
-          op: 'replace',
-          path: 'password',
-          value: password
-        }],
-        description: 'update the password for an invite',
-        service: SERVICE_NAME
-      }
-    )
-  }
-
-  function updateInvitePhoneNumber (inviteCode, phoneNumber) {
-    return baseClient.patch(
-      {
-        baseUrl,
-        url: `/v1/api/invites/${inviteCode}`,
-        json: true,
-        body: [{
-          op: 'replace',
-          path: 'telephone_number',
-          value: phoneNumber
-        }],
-        description: 'update the phone number for an invite',
-        service: SERVICE_NAME
-      }
-    )
-  }
-
-  function sendOtp (inviteCode) {
-    return baseClient.post(
-      {
-        baseUrl,
-        url: `/v1/api/invites/${inviteCode}/send-otp`,
-        json: true,
-        description: 'send OTP code',
-        service: SERVICE_NAME
-      }
-    )
-  }
-
-  function reprovisionOtp (inviteCode) {
-    return baseClient.post(
-      {
-        baseUrl,
-        url: `/v1/api/invites/${inviteCode}/reprovision-otp`,
-        json: true,
-        description: 're-provision OTP key',
-        service: SERVICE_NAME
-      }
-    )
+  async function getValidatedInvite (inviteCode) {
+    const url = `${baseUrl}/v1/api/invites/${inviteCode}`
```

Comment:
I'm assuming that `inviteCode` is from user input (or something the user can manipulate) so it makes my previous comment about encoding more important in this instance.

Reply:
Good point. Please see earlier comment.